workshops/base_views.py

```diff
@@ -206,3 +206,19 @@ def get_success_url(self):
             return next_url
         else:
             return default_url
+
+
+class PrepopulationSupportMixin:
+
+    def get_initial(self):
+        return self.request.GET.dict()
```

Comment:
Can you show an use case where we would not like to have a field prepopulated even though it exists as a GET parameter?
I think we can do that when we arrive at such a use case.